src/components/Table/ListTable/TableAction/ImportTable.vue

```diff
@@ -490,6 +490,9 @@ export default {
 }
 
 .btn-groups {
+  display: flex;
+  justify-content: flex-end;
+
   ::v-deep .el-button.is-disabled {
     cursor: not-allowed;
   }
```

Comment:
There is no code provided to analyze for discrepancies. Please share the problematic lines of code so I can assist you better.